spec/Standard_Distributions.tex

```diff
@@ -1,446 +0,0 @@
-\sekshun{Standard Distributions}
```

Comment:
Additionally, when we've removed sections in the past, often the information in them gets placed in the library documentation, and I worry we'll lose this information.